Packages/vcs/Lib/vtk_ui/textbox.py

```diff
@@ -47,7 +66,11 @@ def add_character(self, character):
         row = rows[self.row]
 
         if self.column >= len(row):
-            row += character
+            if self.blank:
```

Comment:
Allows the user to delete the last character without breaking everything; a lot of the vtk text stuff didn't appreciate empty strings.